checkov/common/bridgecrew/vulnerability_scanning/integrations/package_scanning.py

```diff
@@ -0,0 +1,53 @@
+import os
+from pathlib import Path
+from typing import Dict, Any
+from datetime import datetime, timedelta
+
+from checkov.common.bridgecrew.platform_integration import BcPlatformIntegration
+from checkov.common.bridgecrew.vulnerability_scanning.integrations.twistcli import TwistcliIntegration
+from checkov.common.util.str_utils import removeprefix
+
+
+class PackageScanningIntegration(TwistcliIntegration):
+    # not used for package scanning
+    def _create_report(self, **kwargs: Any) -> Dict[str, Any]:
+        pass
+
+    def _create_report_async(
```

Comment:
@gruebel Can you explain why we need these 2 methods, and they can't be 1?

Reply:
merged them together, I did it to make it more clear, that those private methods are used in a bit different context 🙂